portfolio/src/test/java/com/google/sps/CampaignServletTest.java

```diff
@@ -121,17 +135,17 @@ public void processJSONTest2() {
 
 	/* Return error message with field mask values that were not found in "result" */
```

Comment:
test names should be descriptive enough that such comments should be redundant. please update test names to be more descriptive instead. same everywhere throughout file